mysql-router/Dockerfile

```diff
@@ -55,7 +55,10 @@ COPY dockerdir /
 
 RUN set -ex; \
     mkdir -p /tmp/mysqlrouter/run /var/lib/mysqlrouter ; \
-    chown -R 1001:1001 /tmp/mysqlrouter/run /var/lib/mysqlrouter
+    chown -R 1001:1001 /tmp/mysqlrouter/run /var/lib/mysqlrouter ; \
+    chmod -R 777 /var/log/mysqlrouter ; \
+    chmod -R 777 /tmp/mysqlrouter ; \
+    chmod -R 777 /etc/mysqlrouter
```

Comment:
why did we merge this without resolving @EvgeniyPatlan's comment?

Reply:
@egegunes I noticed that some other Dockerfiles had 777 as the permission , eg: For PS, `chmod 777 /var/run/mysqld`.  Discussed this with @EvgeniyPatlan on Slack and the PR was approved. Sorry, for missing to update it here. Also @egegunes, is it okay for the permission mode to remain 777? If not what would the preferred permission mode be? If so, I shall create another PR and submit the changes related to the permission.